controls/RHEL-07-040190.rb

```diff
@@ -31,8 +31,8 @@
 If “ClientAliveInterval” is not set to “600” in /etc/ ssh/sshd_config, and a lower value is not documented with the Information System Security Officer (ISSO) as an operational requirement, this is a finding.'
 
 # START_DESCRIBE RHEL-07-040190
-  describe file('') do
-    it { should match // }
+  describe sshd_config do
+    its('ClientAliveInterval') { should eq '600' }
```

Comment:
What I was thinking that different companies may have different requirements and it looks like the STIG also has a leeway in there with the allowance for a different value for the timeout. I am just suggesting that if we allow folks to pass in a value they may be able to still run this check as per their policy. I can also push out a PR later to add that in so that we make the check configurable.